src/Components/Patient/SampleTest.tsx

```diff
@@ -66,10 +75,39 @@ export const SampleTest = (props: any) => {
   const { facilityId, patientId } = props;
   const [state, dispatch] = useReducer(sampleTestFormReducer, initialState);
   const [isLoading, setIsLoading] = useState(false);
+  const [facilityName, setFacilityName] = useState<Array<FacilityNameModel>>(
+    []
+  );
 
   const headerText = "Request Sample";
   const buttonText = "Confirm your request to send sample for testing";
 
+  const fetchFacilityName = useCallback(
+    async (status: statusType) => {
+      const facility_type = 950;
+      const FacilityNameList = await dispatchAction(
+        getFacilities({ facility_type })
```

Comment:
Codacy found an issue: [Identifier 'facility_type' is not in camel case.](https://app.codacy.com/gh/coronasafe/care_fe/pullRequest?prid=5828652)